org/cloudfoundry.yml

```diff
@@ -2707,6 +2711,7 @@ orgs:
               noaa: admin
               notifications-release: admin
               operationq: admin
+              pcap-release: admin
```

Comment:
Please don't add anymore projects to non-standard teams. They will be removed rather soon -> #262 

Access is granted by teams generated from WG charter. Admin access is currently only available for the WG leads (this topic is under discussion).

Reply:
done, thx